packages/web-components/package.json

```diff
@@ -47,19 +49,20 @@
     "test-firefox:verbose": "karma start karma.conf.js --browsers=FirefoxHeadless --single-run --coverage --reporter=mocha"
   },
   "devDependencies": {
-    "@microsoft/eslint-config-fast-dna": "^1.2.0",
     "@microsoft/api-extractor": "7.7.1",
-    "@storybook/addons": "^5.3.8",
-    "@storybook/cli": "^5.3.8",
-    "@storybook/html": "^5.3.8",
-    "@storybook/theming": "^5.3.8",
+    "@storybook/addons": "^6.1.17",
+    "@storybook/html": "^6.1.17",
+    "@storybook/theming": "^6.1.17",
     "@types/chai": "^4.2.11",
     "@types/chai-spies": "^1.0.1",
     "@types/karma": "^5.0.0",
     "@types/mocha": "^7.0.2",
+    "@typescript-eslint/eslint-plugin": "4.22.0",
+    "@typescript-eslint/parser": "4.22.0",
```

Comment:
Remove these since they're the same as at the repo root

Reply:
@ecraig12345 I don't see these called out at the root; is this in a nested package perhaps being used by the react projects, etc?

Reply:
Oh sorry, I was looking at either a branch or the wrong package.json

Reply:
No worries, I've left these but reconciled the others. I need to fix a test, but I'll push the updates based on your feedback now as a PR against this branch.

Reply:
PR for the dependency changes: https://github.com/microsoft/fluentui/pull/18822